src/index.js

```diff
@@ -236,9 +236,12 @@ var MaskedInput = React.createClass({
   },
 
   _keyPressPropName() {
-    return navigator.userAgent.match(/Android/i)
+    if (navigator) {
```

Comment:
can you do a `typeof navigator !== 'undefined'` check? strict mode would still throw on this.